src/interfaces/taskDetails.type.ts

```diff
@@ -29,6 +29,9 @@ export type DependencyListProps = {
 export type TaskDetailsProps = {
     detailType: string;
     value?: string;
+    icon_url?: string;
+    icon?: React.ReactElement;
```

Comment:
- are we using the `Details` component at any other place where the icon is different ? 
If not I think we can move the icon inside the component, if there comes a requirement for having different icon we can then introduce this.